core/src/main/csharp/ch/cyberduck/core/CredentialManagerPasswordStore.cs

```diff
@@ -211,27 +221,37 @@ public override void save(Host bookmark)
 
         private static Uri ToUri(Host bookmark)
         {
-            var protocol = bookmark.getProtocol();
-            var credentials = bookmark.getCredentials();
-
-            var targetBuilder = new UriBuilder(PreferencesFactory.get().getProperty("application.container.name"), string.Empty);
+            PasswordStoreDescriptorService service = (PasswordStoreDescriptorService)bookmark.getProtocol().getFeature(typeof(PasswordStorePrefixService));
+            var targetBuilder = new UriBuilder(new HostPreferences(bookmark).getProperty("application.container.name"), string.Empty);
             var pathBuilder = new StringBuilder();
-            pathBuilder.Append(protocol.getIdentifier());
-            if (protocol.isHostnameConfigurable() || !(protocol.isTokenConfigurable() || protocol.isOAuthConfigurable()))
```

Comment:
Is this equivalent to the new checks in `CredentialManagerPasswordStoreDescriptorService` which are different.